src/stratis_cli/_error_codes.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2022 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Error codes
+"""
+# isort: STDLIB
+from enum import IntEnum
+
+
+class PoolMaintenanceErrorCode(IntEnum):
+    """
+    Maintenance error codes for the pool.
+    """
+
+    NO_IPC_REQUESTS = 1
+    NO_POOL_CHANGES = 2
+    READ_ONLY = 3
```

Comment:
This appears to be lingering from when we removed it from stratisd

Reply:
I'm addressing that in #869